js/main.js

```diff
@@ -499,7 +499,8 @@
 		},
 		
 		/*
-		
+		    send the other side "<you> is typing..."
```

Comment:
Capitalise the "Send" please, and if you may, please change <you> to <name> for elegancy.